docs/technical-reference.md

```diff
@@ -256,44 +256,55 @@ users to be able to sign up.
 
 ## Single sign-on (SSO)
 
+> You will need SSO administrator privileges to add the Quilt application.
+
 ### Google
 
 You can enable users on your Google domain to sign in to Quilt.
 Refer to [Google's instructions on OAuth2 user agents](https://developers.google.com/identity/protocols/OAuth2UserAgent)
-and create authorization credentials to identify your Quilt stack to Google's OAuth 2.0 server.
+and create authorization credentials to identify your Quilt stack
+to Google's OAuth 2.0 server.
 
 ![](./imgs/google_console.png)
 
-In the template menu (CloudFormation or Service Catalog), select Google under *User authentication to Quilt*. Enter the domain or domains of your Google apps account under *SingleSignOnDomains*. Enter the *Client ID* of the OAuth 2.0 credentials you created into the field labeled *GoogleClientId*
-
-![](./imgs/google_auth.png)
+Copy the `Client ID` and `Client secret` values to a safe place.
+Once you know your Quilt catalog url, add `[QUILT_CATALOG_URL]/oauth-callback`
+to the **Authorized redirect URIs** section.
 
 ### Active Directory
 
-1. Go to Azure Portal > Active Directory > App Registrations
-1. Click New Registration
-1. Name the app, select the Supported account types
-1. Set a Redirect URI from a "Single-page application" to
-`https://<QuiltWebHost>/oauth-callback`
-1. Once the application has been created you will need both its Application
-(client) ID and Directory (tenant) ID
+1. Go to Azure Portal > Active Directory > App Registrations.
+1. Click "New Registration".
+1. Name the app, select the Supported account types.
+1. Click "Add a platform", "Web", and enter the `Redirect URIs` value
+`[QUILT_CATALOG_URL]/oauth-callback`. Click "Save" at the bottom.
```

Comment:
I found the `[]` brackets confusing. Suggest just `YOUR_QUILT_CATALOG_URL/foo` for everywhere.